linux-anvil-aarch64/Dockerfile

```diff
@@ -16,8 +19,8 @@ ENV LANGUAGE=en_US.UTF-8
 ADD https://loripsum.net/api /opt/docker/etc/gibberish
 
 # Resolves a nasty NOKEY warning that appears when using yum.
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER} && \
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER}-aarch64
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${DISTRO_VER} && \
+    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${DISTRO_VER}-aarch64
```

Comment:
aren't these two line still hyper specific to centos?

Reply:
understood thank you!

Reply:
Also have a follow up PR where we might be able to merge the Dockerfiles together conditioning on `DISTRO_NAME`, etc.. So should make these simpler to manage and update